operator/Makefile

```diff
@@ -116,10 +114,9 @@ $(LOCALBIN):
 	mkdir -p $(LOCALBIN)
 
 # Setting SHELL to bash allows bash commands to be executed by recipes.
-# This is a requirement for 'setup-envtest.sh' in the test target.
 # Options are set to exit when a recipe line exits non-zero or a piped command fails.
-SHELL = /usr/bin/env bash
-.SHELLFLAGS = -o pipefail -euc
+SHELL = /usr/bin/env bash -o pipefail
```

Comment:
Why has the `-o pipefail` been moved from `.SHELLFLAGS` to `SHELL`? And why was `-u` removed?

Reply:
Yeah, how exactly this was evolving was not very clear to me. I meant to come back to this once the bulk of the change was passing, and I forgot ☹️

Reply:
Ok, would be curious about this one.